AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Device.java

```diff
@@ -159,29 +147,24 @@ public int getPhoneID() {
         return mPhoneID;
     }
 
+    private String getSimInformation(String simInfo, String errorMsg) {
+        try {
+            if(simInfo != null && !simInfo.isEmpty())
+                return simInfo;
+        } catch (Exception e) {
+            // SIM methods can cause Exceptions on some devices
+            log.error(String.format("%s %s", errorMsg, e.toString()));
+        }
+        return "N/A";
```

Comment:
you wouldn't use abbreviations for this in danish or japanese either.
and is N/V really used in german? i cannot find any examples of it on the internet or in wahrig.